### PR TITLE
SM-34 footer

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**
You can resize the browser and check if it looks as on screenshots. On screens smaller than 576px footer content should be in one column

image
image
image

### CODE REVIEW

**Review by n0macx — DISMISSED** [review]
Please could You fix accessibility issues - headings and links.

File: src/features/layout/Layout.tsx [hunk 1]
```diff
@@ -7,9 +7,11 @@ const Layout = () => {
   return (
     <>
       <Nav />
-      <div>
-        <Outlet />
-      </div>
+      <main className=" mx-auto flex h-[80vh] flex-col xl:max-w-7xl ">
```
Comment (n0macx) [inline src/features/layout/Layout.tsx]: Maybe we dont need xl: to set max-w ?

File: src/features/layout/footer/Footer.tsx [hunk 2]
```diff
@@ -1,26 +1,41 @@
...
-      <div>
-        <button type="button" onClick={() => changeLanguage('pl')}>
-          pl
-        </button>
-        <button type="button" onClick={() => changeLanguage('en')}>
-          en
-        </button>
-      </div>
+      <footer className="h-fit w-full bg-[#232323] text-white">
+        <div className=" sm:px-3 md:px-6 lg:px-8 mx-auto flex flex-col-reverse justify-between p-14 sm:flex-row xl:max-w-7xl ">
```
Comment (n0macx) [inline src/features/layout/footer/Footer.tsx]: Maybe we dont need xl: to set max-w ?

**Review by n0macx — COMMENTED** [review]
Please, could You add a short description and images ?

**Comment by KatarzynaChmielecka** [comment]
> Please could You fix accessibility issues - headings and links.

headings now are paragraph, FB icon has now aria-label attribute

File: src/features/layout/Layout.tsx (hunk 1 above)
Comment (KatarzynaChmielecka) [reply src/features/layout/Layout.tsx]: it's not pushed yet, but we changed it on call :)

File: src/features/layout/footer/Footer.tsx (hunk 2 above)
Comment (KatarzynaChmielecka) [reply src/features/layout/footer/Footer.tsx]: done

**Review by vieraboschkova — APPROVED** [review]
God job!

**Pull request merged**